pkg/mobile/mobileClient.go

```diff
@@ -66,3 +73,49 @@ func (r *MobileClientRepoImpl) DeleteByName(name string) error {
 func (r *MobileClientRepoImpl) Create(app *v1alpha1.MobileClient) error {
 	return sdk.Create(app)
 }
+
+type MobileWatcher struct {
+	events chan watch.Event
+}
+
+func (w MobileWatcher) Stop() {
+	close(w.events)
+	index := 0
+	for ; index < len(handler.watchers); index++ {
+		if handler.watchers[index] == w {
+			break
+		}
+	}
+	handler.watchers = append(handler.watchers[:index], handler.watchers[index+1:]...)
+}
+
+func (w MobileWatcher) ResultChan() <-chan watch.Event {
+	return w.events
+}
+
+type MobileHandler struct {
+	watchers []MobileWatcher
+}
+
+func (h MobileHandler) Handle(c context.Context, e sdk.Event) error {
+	for index := 0; index < len(h.watchers); index++ {
+		h.watchers[index].events <- watch.Event{
+			Type:   "MobileAppsEvent",
+			Object: nil,
+		}
+	}
+	return nil
+}
+
+func (r *MobileClientRepoImpl) Watch() (watch.Interface, error) {
+	if handler == nil {
+		sdk.Watch("mobile.k8s.io/v1alpha1", "MobileClient", r.namespace, 0)
```

Comment:
what will happen when the last argument is set to `0`? How often will the `handler` being invoked?

Reply:
right, so it will only send event when there is a change, is that right?

Reply:
yes, that is how I understand it

Reply:
cool. thanks